governance/executive-council.md

```diff
@@ -0,0 +1,262 @@
+
+3. **Provides Strategic Insight**: The Executive Council offers perspective and guidance on major strategic decisions to ensure the organization’s long-term success.
+
+4. **Holds the [Executive Director](executive-director) accountable**: The Executive Council holds the Executive Director accountable while also providing high-level strategic and financial guidance and oversight.
+
+The Executive Director is currently a voting member of the Executive Council for items unrelated to their salary and accountability.
+
+The Executive Council holds broad expertise in building and running a scientific / data science nonprofit organization. It is complemented by the [Advisory Council](advisory-council), which offers domain-specific expertise, guidance, and input.
+
+## How the Executive Council works
+
+The pyOpenSci Executive Council operates through open discussions, collaborative
+decision-making, and a commitment to transparency. By leveraging diverse
+perspectives, the Council ensures that its guidance aligns with the organization’s
+[mission, vision, and values](mission-values) while meeting the needs of the
+growing pyOpenSci community.
+
+The Council employs flexible and adaptive practices, such as the **lazy consensus**
+approach, to ensure decisions are made efficiently while fostering thoughtful
+dialogue and collective alignment.
+
+## Executive Council Roles & Responsibilities
+
+Each Executive Council (EC) member dedicates no more than **four to six hours per month** to pyOpenSci activities, including meeting attendance. Executive Council members are expected to fulfill the following responsibilities:
+
+- Attend at least 80% of monthly Executive Council meetings.
+- Participate in asynchronous commitments, which include:
+  - Suggesting agenda items for meetings.
+  - Communicating with the Executive Director (ED) via Slack.
+  - Providing timely reviews of materials and documents.
+  - Connecting the Executive Director with potential funding opportunities.
+- Evaluate the performance of the Executive Director on a yearly basis.
+- Approve expense reports.
+- Take meeting minutes and ensure they are [published in this Handbook](ec-minutes).
+
+The Executive Council is currently led by the Executive Council Chair, who approves the Executive Director’s vacation time off requests and sick leave.
+
+:::{note}
+While travel is not explicitly required, members may have opportunities to attend meetings on behalf of pyOpenSci. Travel may be supported by pyOpenSci if needed.
+:::
+
+
+## How the Executive Council makes decisions
+
+The Executive Council supports the Executive Director by serving as a resource
+for input when needed. However, certain items require explicit Executive Council
+approval.
+
+### Decision-making process
+
+The Executive Council generally follows a **lazy consensus** approach for decision-making:
```

Comment:
this is true, except for the cases listed below that require a vote

Reply:
I think we can flip these sections, starting with 'Decisions requiring a vote' and then go into how decisions are made. 

So

## How the Executive Council makes decisions

The Executive Council is required to review and approve decisions affecting the fiduciary aspects of the organization, this includes mission/vision/values, governance policies, budget, Executive Director compensation, and new significant strategic initiatives. The Executive Council also supports the Executive Director by serving as a resource for input when needed. 

All decisions made by the Executive Council must align with the legal
obligations and policies of pyOpenSci’s fiscal sponsor—[Community Initiatives](https://communityin.org/).
This ensures compliance with overarching governance structures and financial
responsibilities.

### Decisions requiring an Executive Council vote

what we already have in this section

### Decision making process

The topics above require a motion and a vote for approval. Motions and approval can be made in Executive Council meetings or asynchronously via email or Slack. Motions and votes will be noted in the EC meeting minutes.  

## Collaboration with the Executive Director

...
- Strategic decisions about pursuing proposals or writing grants.
- Substantial changes to pyOpenSci programs.

In these cases, *lazy consensus* can be employed. About lazy consensus (noted below where to add that section)